pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_unpublishing-a-module-in-pantheon.adoc

```diff
@@ -0,0 +1,22 @@
+include::../attributes.adoc[]
+[id="unpublishing-a-module-in-pantheon_{context}"]
+= Unpublishing a module in {ProductShortName}
+
+Unpublish a module in {ProductShortName} from the customer portal.
+
+.Prerequisites
+
+* You are assigned the publisher role in {ProductShortName}.
```

Comment:
Same minor notes from above about the 'customer portal', etc., otherwise - looking good!

Reply:
made the changes.